README.md

```diff
@@ -105,3 +105,15 @@ this.store.dispatch(new TraverserActions.UpdateTraverserResource({
     }
 }));
 ```
+
+## Tiles
+
+With `angular-traversal`, we manage the main page view, but it also allows to define small blocks within the current page (named tiles).
```

Comment:
I think the following would be better:

>`angular-traversal` allow to manage the main page view.
>Since version 1.3.0, angular traversal allows to define small blocks within the current page (named tiles).